lib/wit.js

```diff
@@ -20,6 +20,8 @@ function Wit(opts) {
     accessToken, apiVersion, actions, headers, logger, witURL
   } = this.config = Object.freeze(validate(opts));
 
+  this.sessions = {};
```

Comment:
if `sessions` is more of a private obj, would not set it on this -- instead would just write `const _sessions = {}`